### ISSUE TITLE
Schedule links

### ISSUE THREAD

**Issue by sstevens2** [UW-Madison-DataScience/2019-01-14-uwmadison-swc#1]
May need to update the schedule links depend on how instructors want to teach git/python

**Comment by sstevens2** [comment]
@cvoter you said you were going to teach from [these materials](https://uw-madison-datascience.github.io/git-novice-custom/) yes?

@gonuke and @pshriwise, let me know if you would like me to change the python links in the schedule.  If you want to work on the custom parts in a repo hosted by the org (instead of in @kkiesling's version) let me know and I can fork it and give you privs to edit it.  Maybe @kkiesling has a suggestion for this since she taught it last?

**Comment by sstevens2** [comment]
No hurry btw. Mostly wanted to start the discussion and have reminders for myself.

**Comment by kkiesling** [comment]
Here is my custom [python/workflow](https://github.com/kkiesling/python-novice-gapminder-custom) lessons repo. Last meeting I went to, we talked about doing a few things:

- Move this repo over to the DataScience github account
- Separate the python lessons from the workflow lessons (python is lessons 1-15, workflow is 16-end).
- update some of the python lessons to be a bit more in line with the original gapminder lessons
- add more socrative questions / making the students figure out the workflow steps themselves rather than showing them first

**Comment by cvoter** [comment]
Git materials are same as what I'm planning on, thanks for checking!

**Comment by gonuke** [comment]
Here is what I prefer to use: https://uw-madison-aci.github.io/python-novice-gapminder/

I like to hit loops/conditionals before Pandas, but mostly that's what I know....

**Comment by pshriwise** [comment]
@gonuke do you have an estimate of how far you're hoping to get in those lessons?

The python-git workflow lessons are pretty flexible to changes in the python section, but prioritizing a couple topics makes the python-workflow section run more smoothly - is that right @kkiesling?

**Comment by sstevens2** [comment]
Thanks @kkiesling for the list above! 👍  We now have a [Data Science Hub Fork](https://github.com/UW-Madison-DataScience/python-novice-gapminder-custom).  Here is the [rendered version](https://uw-madison-datascience.github.io/python-novice-gapminder-custom/).  I also created [issues](https://github.com/UW-Madison-DataScience/python-novice-gapminder-custom/issues) that should reflect the need for the changes you mentioned. 

@gonuke The version above also has loops and conditionals before pandas.  Let me know if you still want to use the [ACI version](https://uw-madison-aci.github.io/python-novice-gapminder/) or if you are okay with moving forward on this version.

I've added @gonuke, @kkiesling, and @pshriwise as collaborators on the Data Science Hub fork.  We should use issues and pull requests(with someone else reviewing and merging) to communicate about  changes to that repo.  I'll also update the links for python and python+git to refer to this version.

Let me know if you see any potential issues with this setup and we can reassess.
